addons/armaos/functions/fnc_shell_getOpts.sqf

```diff
@@ -0,0 +1,124 @@
+// Parse command string and return set command arguments
+
+// option short form:
+// 'ls -a' or combining multiple arguments 'ls -alh'
+// option long form:
+// 'ls --all -l --human-readable'
+// option with argument:
+// 'ls --width=5'
```

Comment:
At this point width is only an example for an long-form option that takes an argument. I used the `=` sign, because [this](https://man7.org/linux/man-pages/man1/ls.1.html) man page was my template. You can see there that they use the `=` sign. Additionally I thought that the `=` would make parsing options easier. Should we get rid of the `=` sign.